src/app/submissions/submissions.service.ts

```diff
@@ -52,6 +52,23 @@ export class SubmissionsService {
     });
   }
 
+  updateSubmissonNotifcation () {
+    const data = {
+      'message': this.userService.get().name + ' has complete the survey ' + this.submission.parent.name,
+      'link': '/surveys',
```

Comment:
it should have the id as param

Reply:
@lmmrssa we dont have the survey detail page, so i donot add the id as link param. Could you please help me how to manage that. Thank you